gamemodes/core/player/player_items.inc

```diff
@@ -179,8 +179,8 @@ static void:_SaveItem(playerid, const itemName[], value) {
         "UPDATE player_items SET ? = ? + ? WHERE u_id = ?"
     );
 
-    MySQL_Bind(stmt_saveItem, 0, itemName, true);
-    MySQL_Bind(stmt_saveItem, 1, itemName, true);
+    MySQL_Bind(stmt_saveItem, 0, itemName);
+    MySQL_Bind(stmt_saveItem, 1, itemName);
```

Comment:
this is needed to allow raw string to be the input

Reply:
When i compiled it, gives me an error but gonna fix it

Reply:
need to update package ``sampctl p ensure`` because I've recently updated the prepared statement library.